app/forms/openid_connect_authorize_form.rb

```diff
@@ -214,7 +214,7 @@ def result_uri
   end
 
   def error_redirect_uri
-    uri = redirect_uri unless errors.include?(:redirect_uri)
+    uri = redirect_uri unless (errors.include?(:redirect_uri) || errors.include?(:client_id))
```

Comment:
ditto: `unless` + compound statement is my mortal enemy
```suggestion
    return if errors.include?(:redirect_uri) || errors.include?(:client_id)
    uri = redirect_uri
```

Reply:
good call, resolved